### ISSUE TITLE
My List Actions don't receive keyboard focus on small screens

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/dimes#850]
## Describe the bug
Accessibility issue 124: On the My List page on small screens, the menu items in the "Actions" dropdown menu do not receive keyboard focus, accept the "Schedule a Visit" link. This blocks access to those options for some users, and is a violation of WCAG 2.1.1 Keyboard.

## To reproduce
1. Go to the My List page on a 320px-width screen
2. Tab to the "Actions" button and hit "Return" to open it
3. Tab down the dropdown menu
4. See that only the first menu item receives keyboard focus

## Impact on your work
This blocks access to users trying to access content on the mobile site via keyboard or related assistive tech.

**Comment by HaSistrunk** [comment]
This is likely related to issue #611 . Looking at this comprehensively with the NavDropdown implementation of the dropdown, I would suggest replacing `react-accessible-dropdown-menu-hook` with a custom component. We don't need the complexity of an aria menu and menuitems in our implementation, and because we are mixing buttons and links in the dropdown items it's causing problems that we'll have to work around anyway outside of its standard implementation to ensure accessibility.

I think we can create a dropdown disclosure pattern with a list of links/buttons and `display:"none"` applied to that list when it's not open to prevent keyboard and screen reader access until it's open.

**Comment by helrond** [comment]
closed by #859
